ivadomed/utils.py

```diff
@@ -6,6 +6,7 @@
 import numpy as np
 import torch
 import torch.nn as nn
+from medicaltorch.datasets import MRI2DSegmentationDataset
```

Comment:
Instead of `BidsDataset` because we can't assume a BIDS dataset.

Reply:
will this create some retro-compatibility when using it with BIDS datasets? or lack of performance/clarity in the code?

Reply:
Not sure to understand the question but I will try, please let me know if not.

This import is only used by `segment_volume` which deals with only one filename --> compatible with BIDS or no-BIDS data.
Whereas our training pipeline is assuming BIDS datasets --> it uses `BidsDataset` ([here](https://github.com/neuropoly/ivado-medical-imaging/blob/master/ivadomed/loader.py#L49)) which inherits from `MRI2DSegmentationDataset`.

This is in line with the debate we had this morning / evening in meeting --> how `ivadomed` and `medicaltorch` are intertwined.